docs/getstarted/evaluation.md

```diff
@@ -23,9 +24,9 @@ os.environ["OPENAI_API_KEY"] = "your-openai-key"
 
 Ragas performs a `ground_truth` free evaluation of your RAG pipelines. This is because for most people building a gold labeled dataset which represents in the distribution they get in production is a very expensive process.
 
-```{note}
+:::{note}
```

Comment:
Is this story really required here in getting started? Metrices are their divisions are explained in core concepts.

Reply:
removed